bridge/src/client/cli/client_command.rs

```diff
@@ -79,6 +79,30 @@ impl ClientCommand {
         }
     }
 
+    async fn get_funding_utxo_input(&self, utxo_arg: Option<&String>) -> io::Result<Input> {
+        let utxo = utxo_arg.expect("Missing UTXO argument, please see help.");
+        let outpoint = OutPoint::from_str(utxo).expect(
+            "Could not parse the provided UTXO, please see help for the correct format.",
+        );
+        let tx = self
```

Comment:
Let's change the name so it's not duplicated with the other variable to e.g. tx_res.
```suggestion
        let tx_res = self
```

Reply:
This is intentional. It makes use of the Rust's variable shadowing feature. We do it in a couple other places in the project. It's actually a recommended way to "spare us from having to come up with different names" according to the Rust book: https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html?highlight=shadow#shadowing.